update_chrome.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+FILE="/Users/will/Library/Application Support/Google/Chrome/Local State"
+
+echo "Closing Chrome..."
+pkill -f "Google Chrome" 2>/dev/null
+sleep 2
+
+echo "Backing up..."
+cp "$FILE" ~/Desktop/Local_State_$(date +%Y%m%d_%H%M).backup
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)